### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by zadilkhwaja** [EddieHubCommunity/support#5383] [Q&A] [answered]
I would like to do code review and be a part of the maintainer team for this repo (awesome-github-profiles). I've joined the EddieHubCommunity, can i know what are the next steps? @Panquesito7

**Comment by vedanshiprajapati** [comment, marked as answer]
hey, to be the maintainer ..you have to review the pull requests constantly for a while..then after sometime if you approve the right changes and suggest changes that are good for the repository ..that will be enough for you to become maintainer ..other maintainer will notice your work and eventually you'll be one:)

**Reply by zadilkhwaja** [reply]
> hey, to be the maintainer ..you have to review the pull requests constantly for a while..then after sometime if you approve the right changes and suggest changes that are good for the repository ..that will be enough for you to become maintainer ..other maintainer will notice your work and eventually you'll be one:)

ok thanks!
